scripts/ci/tags.yaml

```diff
@@ -348,3 +348,41 @@ Sidewalk:
     - modules/lib/zcbor/
     - bootloader/mcuboot/
     - sidewalk/
+
+find_my:
+  files:
```

Comment:
We seem to be missing a few dependencies for the Thingy:53 target

zephyr/drivers/sensor/adi/adxl362
zephyr/drivers/sensor/bosch/bme680
zephyr/drivers/sensor/bosch/bmi270
zephyr/drivers/sensor/bosch/bmm150
zephyr/drivers/sensor/voltage_divider

Reply:
Added, thanks